docs/app/partials/layout-introduction.tmpl.html

```diff
@@ -0,0 +1,364 @@
+<div ng-controller="LayoutCtrl" class="layout-content" ng-cloak>
+
+  <h3>Overview</h3>
+  <p>
+    Angular Material's Layout features are purposed as syntactic HTML/CSS sugar to allow developers
```

Comment:
For the rest of this section, I would use the term *"layout directive"* vs. "layout attribute". I think this makes it much more clear that we have code that's actually doing stuff, vs. just targeting the attributes with CSS.